components/button.json

```diff
@@ -18,7 +18,8 @@
       },
       "#disabled": {
         "background": "@theme-button-primary-disabled",
-        "text": "@theme-text-primary-disabled"
+        "text": "@theme-text-primary-disabled",
+        "border": "@theme-button-border-disabled"
```

Comment:
the border being '@theme-button-border-disabled', however, is not true of non-hc themes right?